projects/igniteui-angular/src/lib/core/styles/components/grid/_grid-theme.scss

```diff
@@ -88,7 +123,7 @@
     }
 
     @if not($row-selected-text-color) and $row-selected-background {
-        $row-selected-text-color: text-contrast($row-selected-background);
+        $row-selected-text-color: $grouparea-background;
```

Comment:
Why? What is the relationship between the row-selected-text-color and the grouparea-background color? Do we always assume the color of the grouparea-background for the text color of the selected row? What if they are the same?

Reply:
This is by mistake, thanks for catching it